src/main/edit_delete/JSONTranscription.ts

```diff
@@ -0,0 +1,12 @@
+import { Transcription, Word } from 'sharedTypes';
+
+export default class JSONTranscription implements Transcription {
```

Comment:
Same idea, in this case instead of calling `new JSONTranscription` we can just make an object like `({ confidence, words })` and pass that around, unless there's something specific this class needs to do (and even then could just be simpler to have a top level function that takes a Transcription as argument and returns whatever is needed, simpler and purer that way)